<!DOCTYPE HTML>
<html>
	<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<title>Jenny Chen</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="description" content="" />
	<meta name="keywords" content="" />
	<meta name="author" content="" />

  <!-- Facebook and Twitter integration -->
	<meta property="og:title" content=""/>
	<meta property="og:image" content=""/>
	<meta property="og:url" content=""/>
	<meta property="og:site_name" content=""/>
	<meta property="og:description" content=""/>
	<meta name="twitter:title" content="" />
	<meta name="twitter:image" content="" />
	<meta name="twitter:url" content="" />
	<meta name="twitter:card" content="" />

	<!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
	<link rel="shortcut icon" href="favicon.ico">

	<link href="https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,700" rel="stylesheet">
	
	<!-- Animate.css -->
	<link rel="stylesheet" href="css/animate.css">
	<!-- Icomoon Icon Fonts-->
	<link rel="stylesheet" href="css/icomoon.css">
	<!-- Bootstrap  -->
	<link rel="stylesheet" href="css/bootstrap.css">
	<!-- Flexslider  -->
	<link rel="stylesheet" href="css/flexslider.css">
	<!-- Flaticons  -->
	<link rel="stylesheet" href="fonts/flaticon/font/flaticon.css">
	<!-- Owl Carousel -->
	<link rel="stylesheet" href="css/owl.carousel.min.css">
	<link rel="stylesheet" href="css/owl.theme.default.min.css">
	<!-- Theme style  -->
	<link rel="stylesheet" href="css/style.css">

	<!-- Modernizr JS -->
	<script src="js/modernizr-2.6.2.min.js"></script>
	<!-- FOR IE9 below -->
	<!--[if lt IE 9]>
	<script src="js/respond.min.js"></script>
	<![endif]-->

	</head>
	<body>
	<div id="colorlib-page">
		<div class="container-wrap">
		<a href="#" class="js-colorlib-nav-toggle colorlib-nav-toggle" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><i></i></a>
		<aside id="colorlib-aside" role="complementary" class="border js-fullheight">
			<div class="text-center">
				<div class="author-img" style="background-image: url(images/Jenny_Chen_Hoekstra_Eddy.jpg);"></div>
				<h1 id="colorlib-logo"><a href="index.html">Jenny Chen, PhD</a></h1>
				<span class="position"><a href="#">MOSAIC K99/R00 Postdoctoral Fellow</a><br>Harvard University</span>
			</div>
			<nav id="colorlib-main-menu" role="navigation" class="navbar">
				<div id="navbar" class="collapse">
					<ul>
						<li class = "active"><a href="#" data-nav-section="about">About</a></li>
						<li><a href="#" data-nav-section="projects">Projects</a></li>
						<li><a href="#" data-nav-section="past">Past</a></li>
						<li><a href="#" data-nav-section="education">Education</a></li>
						<li><a href="#" data-nav-section="contact">Contact</a></li>

					</ul>
				</div>
			</nav>

			<div class="text-center">
					<a href="https://twitter.com/jjennychenn" target="_blank"><i class="icon-twitter2"></i></a>&nbsp;&nbsp;
					<a href="https://scholar.google.com/citations?user=QIvEOakAAAAJ&hl=en" target="_blank"><i class="icon-google2"></i></a>&nbsp;&nbsp;
					<a href="https://github.com/jjenny" target="_blank"><i class="icon-github"></i></a>
			</div>
			
			<br>
                        <br>
                        <br>
			<div class="colorlib-footer">	
				<p><small><!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved
<br>Template by <a href="https://colorlib.com" target="_blank">Colorlib</a> 
<!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. --> </span> 
</small></p>


			</div>

		</aside>

		<div id="colorlib-main">

			<section class="colorlib-about" data-section="about">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-12">
							<div class="row row-bottom-padded-sm animate-box" data-animate-effect="fadeInLeft">
								<div class="col-md-12">
									<div class="about-desc">
										<span class="heading-meta">About Me</span>
										<h2 class="colorlib-heading">Who Am I?</h2>
										<p><strong>Hi, I'm Jenny Chen.</strong> I am interested in understanding how genes encode for social behavioral traits. I take advantage of deer mice of the genus <em>Peromyscus</em> where monogamy has independently evolved multiple times to investigate the molecular mechanisms underlying mating and parental care behaviors. I use techniques spanning classical linkage studies, animal ethology, comparative genomics, and single-cell transcriptomics. </p>
										<p>I am currently a <a href="https://www.nimhd.nih.gov/programs/extramural/training-career-dev/mosaic-program.html" target="_blank">MOSAIC K99/R00 scholar</a> and formerly a <a href="https://datascience.harvard.edu/" target="_blank">Harvard Data Science Fellow</a>. My work is a collaboration between the <a href="https://hoekstra.oeb.harvard.edu/" target = "_blank">Hoekstra Lab</a> and the <a href="http://eddylab.org/" target="_blank">Eddy Lab</a>. Prior to this, I received my PhD from MIT where I was advised by <a href="https://www.gene.com/scientists/our-scientists/aviv-regev" target="_blank">Aviv Regev</a>. My publications are listed on my <a href="https://scholar.google.com/citations?user=QIvEOakAAAAJ&hl=en" target="_blank">Google Scholar</a> profile and my codebase is available through <a href="https://github.com/jjenny" target="_blank">github</a>.</p>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>

			<section class="colorlib-education" data-section="projects">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<span class="heading-meta">Projects</span>
							<h2 class="colorlib-heading animate-box">Current projects</h2>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12 animate-box" data-animate-effect="fadeInLeft">
							<div class="fancy-collapse-panel">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingOne">
									        <h4 class="panel-title">
									            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne">Peromyscus Brain Expression Atlas</a>
									        </h4>
									    </div>
									    <div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
									         <div class="panel-body">
									         <p><img src="images/brain_atlas.png" width=400></p>
											 <p>
											 The evolution of innate behaviors is in part due to genetic variation acting in the nervous system. Gene regulation may be particularly important because it can evolve in a modular brain-region specific fashion through the concerted action of <em>cis-</em> and <em>trans-</em>regulatory changes. To investigate transcriptional variation and its regulatory basis across the brain, we perform RNA-seq on ten different brain subregions in two sister species of deer mice (<em>Peromyscus maniculatus</em> and <em>P. polionotus</em>) - which differ in a range of innate behaviors, including their social system - and their F1 hybrids. We find that interspecific differential expression is pervasive but varies considerably across brain regions. Through analysis of F1 hybrids, we find that much of this modularity is due to <em>cis-</em>regulatory divergence. Together, these results highlight the modularity of gene expression differences and divergence in the brain, which may be key to explain how the evolution of brain gene expression can contribute to the astonishing diversity of animal behaviors. </p>
											 
											 
									         <p>Paper: <a href="https://www.biorxiv.org/content/10.1101/2023.09.28.559844v1" target="_blank">Evolution of gene expression across brain regions in behaviorally divergent deer mice</a>. <em>bioRxiv</em>, Sept 2023. </p>
            
									         </div>
									    </div>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>


			<section class="colorlib-education" data-section="past">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<span class="heading-meta">Past</span>
							<h2 class="colorlib-heading animate-box">Published projects</h2>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12 animate-box" data-animate-effect="fadeInLeft">
							<div class="fancy-collapse-panel">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingTwo">
									        <h4 class="panel-title">
									            <a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">Mammalian expression evolution</a>
									        </h4>
									    </div>
									    <div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
									         <div class="panel-body">
									         <p><img src="images/mammals.meanexp.png" width=400></p>
											 <p>The evolutionary history of a gene is frequently used to predict its function and medical relevance. However, current comparative genomics methods focus primarily on sequence conservation, and few methods exist for interpreting comparative expression data. Using RNA-sequencing data from 7 tissues across 17 mammalian species, I showed that expression evolution across mammals follows a mathematical model called the Ornstein-Uhlenbeck (OU) process. I demonstrate how to use this model to identify expression pathways underlying conserved and lineage-specific phenotypes. Furthermore, I show how to use this model to estimate the distribution of each gene's evolutionarily optimal expression level in a given tissue. This distribution can then be used to detect deleterious expression levels (i.e. levels outside the optimal expression distribution) in patient expression data. This work provides a statistical framework for interpreting expression data across species and in disease.</p>
									         <p>Paper: <a href="https://genome.cshlp.org/content/29/1/53" target="_blank">A quantitative model for characterizing the evolutionary history of mammalian gene<br>&emsp;&emsp;&emsp;&ensp;expression</a>. <em>Genome Research</em>, Jan 2019. </p>
            
									         </div>
									    </div>
									</div>
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingThree">
									        <h4 class="panel-title">
									            <a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">Long noncoding RNA evolution
									            </a>
									        </h4>
									    </div>
									    <div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
									        <div class="panel-body">
									        <p><img src="images/lncrnas.png"></p>
									         <p>The development of RNA-sequencing technology led to the discovery that thousands, of long noncoding RNAs (lncRNAs) are pervasively transcribed in the mammalian genome. Yet, the function of these genes are unknown. In a collaboration with <a href="http://garberlab.umassmed.edu/" target="_blank">Manuel Garber (UMass Medical)</a>, we developed a computational tool, <a href="http://slncky.github.io/" target="_blank"><em>slncky</em></a>, that identifies lncRNAs from RNA-sequencing data and examines the evolutionary history of these genes. Our tool discovers orthologous lncRNAs and calculates metrics relevant for noncoding transcript evolution such as transcript-transcript identity (what percentage of the transcript sequence is transcribed in another species) and splice site conservation. Our analysis revealed distinct patterns of selection, uncovering several classes of lncRNAs that likely each have distinct functions.  Out of the tens and thousands of currently annotated lncRNAs, we identify 233 constrained lncRNAs which are browsable through the <a href="https://slncky.umassmed.edu/" target="_blank"><em>slncky</em> Evolution Browser</a>.</p>
									         <p>Paper: <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0880-9" target="_blank">Evolutionary analysis across mammals reveals distinct classes of long non-coding RNAs</a>.<br>&emsp;&emsp;&emsp;<em>Genome Biology</em>, January 2016. </p>
									        </div>
									    </div>
									</div>
									
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>

			<section class="colorlib-experience" data-section="education">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<h2 class="colorlib-heading animate-box">Education</h2>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12">
				         <div class="timeline-centered">

					         <article class="timeline-entry animate-box" data-animate-effect="fadeInLeft">
					            <div class="timeline-entry-inner">

					               <div class="timeline-icon color-3">
					                  <i class="icon-pen2"></i>
					               </div>

					               <div class="timeline-label">
					                  <h2><a href="#">PhD in Bioinformatics and Integrative Genomics, Division of Health Sciences and Technology, MIT</a> <span>2011-2018</span></h2>
					                  <p><b>Thesis:</b> Evolutionary signatures for unearthing functional elements in the human transcriptome
									  <br><b>Faculty advisor:</b> Aviv Regev
					               </div>
					            </div>
					         </article>


					         <article class="timeline-entry animate-box" data-animate-effect="fadeInRight">
					            <div class="timeline-entry-inner">
					               <div class="timeline-icon color-1">
					                  <i class="icon-pen2"></i>
					               </div>
					               <div class="timeline-label">
					               	<h2><a href="#">BS/MS in Biomedical Computation, Stanford University</a> <span>2006-2011</span></h2>
					                  <p><b>Thesis:</b> Analysis of Transcription Factor ChIP-Seq Datasets 
									  <br><b>Faculty advisor:</b> Gill Bejerano
					               </div>
					            </div>
					         </article>

					      </div>
					   </div>
				   </div>
				</div>
			</section>

			<section class="colorlib-contact" data-section="contact">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<span class="heading-meta">Contact</span>
							<h2 class="colorlib-heading">Get in touch</h2>
						</div>
					</div>
					
					<div class="row">
						<div class="col-md-5">
							<div class="colorlib-feature colorlib-feature-sm animate-box" data-animate-effect="fadeInLeft">
								<div class="colorlib-icon">
									<i class="icon-mail"></i>
								</div>
								<div class="colorlib-text">
									<p>jennifer underscore chen at fas dot harvard dot edu</p>
								</div>
							</div>

							<div class="colorlib-feature colorlib-feature-sm animate-box" data-animate-effect="fadeInLeft">
								<div class="colorlib-icon">
									<i class="icon-map"></i>
								</div>
								<div class="colorlib-text">
									<p>Rm 1008, 16 Divinity Avenue <br> Cambridge, MA</p>
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>

		</div><!-- end:colorlib-main -->
	</div><!-- end:container-wrap -->
	</div><!-- end:colorlib-page -->

	<!-- jQuery -->
	<script src="js/jquery.min.js"></script>
	<!-- jQuery Easing -->
	<script src="js/jquery.easing.1.3.js"></script>
	<!-- Bootstrap -->
	<script src="js/bootstrap.min.js"></script>
	<!-- Waypoints -->
	<script src="js/jquery.waypoints.min.js"></script>
	<!-- Flexslider -->
	<script src="js/jquery.flexslider-min.js"></script>
	<!-- Owl carousel -->
	<script src="js/owl.carousel.min.js"></script>
	<!-- Counters -->
	<script src="js/jquery.countTo.js"></script>
	
	
	<!-- MAIN JS -->
	<script src="js/main.js"></script>

	</body>
</html>

