<!DOCTYPE HTML>
<html>
	<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<title>Jenny Chen</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="description" content="" />
	<meta name="keywords" content="" />
	<meta name="author" content="" />

  <!-- Facebook and Twitter integration -->
	<meta property="og:title" content=""/>
	<meta property="og:image" content=""/>
	<meta property="og:url" content=""/>
	<meta property="og:site_name" content=""/>
	<meta property="og:description" content=""/>
	<meta name="twitter:title" content="" />
	<meta name="twitter:image" content="" />
	<meta name="twitter:url" content="" />
	<meta name="twitter:card" content="" />

	<!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
	<link rel="shortcut icon" href="favicon.ico">

	<link href="https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,700" rel="stylesheet">
	
	<!-- Animate.css -->
	<link rel="stylesheet" href="css/animate.css">
	<!-- Icomoon Icon Fonts-->
	<link rel="stylesheet" href="css/icomoon.css">
	<!-- Bootstrap  -->
	<link rel="stylesheet" href="css/bootstrap.css">
	<!-- Flexslider  -->
	<link rel="stylesheet" href="css/flexslider.css">
	<!-- Flaticons  -->
	<link rel="stylesheet" href="fonts/flaticon/font/flaticon.css">
	<!-- Owl Carousel -->
	<link rel="stylesheet" href="css/owl.carousel.min.css">
	<link rel="stylesheet" href="css/owl.theme.default.min.css">
	<!-- Theme style  -->
	<link rel="stylesheet" href="css/style.css">

	<!-- Modernizr JS -->
	<script src="js/modernizr-2.6.2.min.js"></script>
	<!-- FOR IE9 below -->
	<!--[if lt IE 9]>
	<script src="js/respond.min.js"></script>
	<![endif]-->

	</head>
	<body>
	<div id="colorlib-page">
		<div class="container-wrap">
		<a href="#" class="js-colorlib-nav-toggle colorlib-nav-toggle" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><i></i></a>
		<aside id="colorlib-aside" role="complementary" class="border js-fullheight">
			<div class="text-center">
				<div class="author-img" style="background-image: url(images/Jenny_Chen_Hoekstra_Eddy.jpg);"></div>
				<h1 id="colorlib-logo"><a href="index.html">Jenny Chen, PhD</a></h1>
				<span class="position"><a href="#">NIH MOSAIC K99/R00 Postdoctoral Fellow</a><br>Harvard University</span>
			</div>
			<nav id="colorlib-main-menu" role="navigation" class="navbar">
				<div id="navbar" class="collapse">
					<ul>
						<li class = "active"><a href="#" data-nav-section="about">About</a></li>
						<li><a href="#" data-nav-section="projects">Projects</a></li>
						<li><a href="#" data-nav-section="past">Past</a></li>
						<li><a href="#" data-nav-section="education">Education</a></li>
						<li><a href="#" data-nav-section="contact">Contact</a></li>

					</ul>
				</div>
			</nav>

			<div class="text-center">
					<a href="https://twitter.com/jjennychenn" target="_blank"><i class="icon-twitter2"></i></a>&nbsp;&nbsp;
					<a href="https://scholar.google.com/citations?user=QIvEOakAAAAJ&hl=en" target="_blank"><i class="icon-google2"></i></a>&nbsp;&nbsp;
					<a href="https://github.com/jjenny" target="_blank"><i class="icon-github"></i></a>
			</div>
			
			<br>
                        <br>
                        <br>
			<div class="colorlib-footer">	
				<p><small><!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved
<br>Template by <a href="https://colorlib.com" target="_blank">Colorlib</a> 
<!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. --> </span> 
</small></p>


			</div>

		</aside>

		<div id="colorlib-main">
			<section id="colorlib-hero" class="js-fullheight" data-section="home">
				<div class="flexslider js-fullheight">
					<ul class="slides">
				   	<li style="background-image: url(images/none.gif);">
				   		<div class="overlay"></div>
				   		<div class="container-fluid">
				   			<div class="row">
					   			<div class="col-md-6 col-md-offset-3 col-md-pull-3 col-sm-12 col-xs-12 js-fullheight slider-text">
					   				<div class="slider-text-inner js-fullheight">
					   					<div class="desc">
											<img src="images/genome_research.gif">
						   					<h1>Jenny Chen, <br>PhD</h1>
						   					<h2>NIH MOSAIC K99/R00 Postdoctoral Fellow</h2>
												<p><a class="btn btn-primary btn-learn" href="files/Jenny_chen_CV.pdf" target="_blank">Download CV <i class="icon-download4"></i></a></p>
											</div>
					   				</div>
					   			</div>
					   		</div>
				   		</div>
				   	</li>
				  	</ul>
			  	</div>
			</section>
			<section class="colorlib-about" data-section="about">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-12">
							<div class="row row-bottom-padded-sm animate-box" data-animate-effect="fadeInLeft">
								<div class="col-md-12">
									<div class="about-desc">
										<span class="heading-meta">About Me</span>
										<h2 class="colorlib-heading">Who Am I?</h2>
										<p><strong>Hi, I'm Jenny Chen.</strong> I am interested in understanding how genes encode for behavioral traits. The existence of innate, heritable animal behaviors suggests that behavior can be specified by genes, but few behavioral genes are known, and even less is known about the mechanism by which they act. I take advantage of evolutionary patterns of innate animal behaviors combined with comparative genomics, single cell transcriptomics, and other large-scale sequencing approaches to investigate how genes govern behavior. Currently, I am investigating the genetic basis of reproductive behaviors across two sister species of <em>Peromyscus</em> deer mice that lie on opposite ends of the monogamy-promiscuity spectrum and exhibit divergent parenting strategies.
										<!--
										<table><td><img src="images/po_bw.png" width=400></img></td><td><b>Left:</b> A <em>P. polionotus</em> father who has made a nest and is huddling over his pup. <br><br><b>Right:</b> A <em>P. maniculatus</em> father who has left his nestlet and his pup untouched.</td>
										</table>
										--!>
										</p>
										<p>I am an <a href="https://www.nimhd.nih.gov/programs/extramural/training-career-dev/mosaic-program.html" target="_blank">NIH MOSAIC K99/R00 scholar</a> and formerly a <a href="https://datascience.harvard.edu/" target="_blank">Harvard Data Science Fellow</a>. My work is a collaboration between the <a href="https://hoekstra.oeb.harvard.edu/" target = "_blank">Hoekstra Lab</a> and the <a href="http://eddylab.org/" target="_blank">Eddy Lab</a> at Harvard University. Prior to this, I received my PhD in Bioinformatics and Integrative Genomics from MIT where I was advised by <a href="https://www.gene.com/scientists/our-scientists/aviv-regev" target="_blank">Aviv Regev</a>.</p>
										<p>You can find publications on my <a href="https://scholar.google.com/citations?user=QIvEOakAAAAJ&hl=en" target="_blank">Google Scholar</a> profile and my codebase on through <a href="https://github.com/jjenny" target="_blank">github</a>.</p>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>

			<section class="colorlib-education" data-section="projects">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<span class="heading-meta">Projects</span>
							<h2 class="colorlib-heading animate-box">Current project</h2>
							<h3>Investigating the genetic basis of reproductive behaviors in <em>Peromyscus</em> deer mice</h3>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12 animate-box" data-animate-effect="fadeInLeft">
							<div class="fancy-collapse-panel">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingOne">
									        <h4 class="panel-title">
									            <a data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne">Cellular evolution of the hypothalamic preoptic area</a>
									        </h4>
									    </div>
									    <div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
									         <div class="panel-body">
									         <p><img src="images/mpoa_fig.png" width=750></p>
											 <p>
											I investigated the evolution of the hypothalamic medial preoptic area (MPOA), a brain region that plays a critical role in social behavior, across two sister species of deer mice (<em>P. maniculatus</em> and <em>P. polionotus</em>) with divergent reproductive behaviors. <em>P. maniculatus</em> mate promiscuously and only mothers exhibit parental care while <em>P. polionotus</em> produce offspring with the same mate across their lifetime and both fathers and mathers provide parental care.
											<br><br>
											Using single-nucleus RNA-sequencing, I built a cellular atlas of the MPOA for males and females of both <em>Peromyscus</em> species and asked how neuronal cell abundance and gene expression differences may be contributing to behavior evolution. This study revealed three biological findings. First, I observed a substantial reduction in sex-biased gene expression in the monogamous <em>P. polionotus</em> compared to the promiscuous <em>P. maniculatus</em>. While morphological traits such as body size and biological ornamentation are well known to be less sexually dimorphic in monogamous animals, to our knowledge, this is the first genome-wide observation of a molecular trait (i.e. neuronal gene expression) being less sexually dimorphic in a monogamous than promiscuous species. Second, I identified four cell types differentially abundant across species, two of which -- vasopressin arginine (AVP) and galanin (GAL) neurons -- may account for differential levels of parenting behavior based on known functions of these neurons. Third, I identified two cell types that are sex-biased in abundance and highly enriched for sex-biased genes in both species, suggesting these cell types are crucial for sex-specific behaviors. Notably, I found a striking enrichment of sex-biased genes to be neuropeptides.
											<br><br>
											This work suggests the genetic control of mating and parenting behaviors may involve changes in both cell abundance and sex-biased expression, with neuropeptide regulation likely playing a crucial role.
											</p> 
											 
									         <p>Paper: <a href="https://www.biorxiv.org/content/10.1101/2024.08.22.608850v1" target="_blank">Cellular evolution of the hypothalamic preoptic area of behaviorally divergent deer mice</a>. <em>bioRxiv</em>, Aug 2024. </p>
            
									         </div>
									    </div>
									</div>
								</div>
							</div>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12 animate-box" data-animate-effect="fadeInRight">
							<div class="fancy-collapse-panel">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingOne">
									        <h4 class="panel-title">
									            <a data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">Peromyscus Brain Expression Atlas</a>
									        </h4>
									    </div>
									    <div id="collapseFour" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
									         <div class="panel-body">
									         <p><img src="images/brain_atlas.png" width=750></p>
											 <p>
											 Gene regulation has been proposed to be particularly important to the evolution of innate behaviors because it can evolve in a modular brain-region specific fashion through the concerted action of <em>cis-</em> and <em>trans-</em>regulatory changes. To investigate transcriptional variation and its regulatory basis across the brain, we performed RNA-seq on ten different brain subregions in <em>P. maniculatus</em> and <em>P. polionotus</em> and their F1 hybrids. Comparing species expression differences to allelic expression differences in the F1 hybrids allowed us to dissect the <em>cis-</em> and <em>trans-</em>regulatory components of expression divergence, as both alleles in F1 hybrids share the same <em>trans-</em>regulatory environment. 
											 <br><br>We found that interspecific differential expression is pervasive but varies considerably across brain regions. Through analysis of F1 hybrids, we found that much of this modularity is due to <em>cis-</em>regulatory divergence. This work suggests the propensity of <em>cis-</em>regulatory changes across closely related species may provide important substrates for modulating specific behavioral traits at short evolutionary time scales. </p>
											 
											 
									         <p>Paper: <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.17270" target="_blank">Evolution of gene expression across brain regions in behaviorally divergent deer mice</a>. <em>Molecular Ecology</em>, Jan 2024. </p>
            
									         </div>
									    </div>
									</div>
								</div>
							</div>
						</div>
					</div>
					<br>
					<br>
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<h2 class="colorlib-heading animate-box">Current project</h2>
							<h3>Discovery of novel neuropetides</h3>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12 animate-box" data-animate-effect="fadeInLeft">
							<div class="fancy-collapse-panel">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingOne">
									        <h4 class="panel-title">
									            <a data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive">A putative novel neuropeptide involved in reproductive behavior</a>
									        </h4>
									    </div>
									    <div id="collapseFive" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
									         <div class="panel-body">
											 <img src="images/neuropeptide.png" width=750>
									         Neuropeptides are potent signaling molecules that play vital roles ranging from regulating sleep to feeding behavior to social behavior. My postdoctoral work further suggests neuropeptides are important evolutionary tuning dials. While over 100 neuropeptides are known, more likely await discovery: their small size and rapid evolution hinder bioinformatics identification, and those that are expressed in very specific cell types are challenging to detect by bulk RNA-seq or mass spectrometry. I developed a computational screen to uncover new neuropeptides conserved across vertebrates by using advancements in machine learning and large language models to identify small, secreted peptides and single-cell sequencing data to identify genes expressed in specific neuronal cell populations. I have discovered one putative neuropeptide that appears to impact fertility when knocked out of rodent animal models. This work is ongoing.
											 </div>
									    </div>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>


			<section class="colorlib-education" data-section="past">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<span class="heading-meta">Past</span>
							<h2 class="colorlib-heading animate-box">Past projects</h2>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12 animate-box" data-animate-effect="fadeInLeft">
							<div class="fancy-collapse-panel">
								<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingTwo">
									        <h4 class="panel-title">
									            <a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">Mammalian expression evolution</a>
									        </h4>
									    </div>
									    <div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
									         <div class="panel-body">
											 <img src="images/mammals.png" width=650>

											 <br><br>Many methods exist for using comparative sequence data to infer gene function, such as using the extent of sequence conservation to predict essentiality. However, few methods exist to do the same for cross-species expression data. Here, I developed a method that uses a model of continuous trait evolution, the Ornstein-Uhlenbeck process, to estimate the evolutionary optimal distribution of a gene's expression from RNA-seq data across 17 mammals. I showed that these distributions give us clues to a gene's function. For example, essential genes have evolutionary distributions with very small variances, suggesting there is strong selective pressure for these genes to be expressed at very precise expression levels. Conversely, secreted genes have large variances. I also demonstrated the power of this distribution to infer function from expression patterns, such as detecting expression pathways under directional selection that may underlie lineage-specific phenotypes, or identifying deleterious expression of disease-causing genes from patient data. This work provides a statistical framework for interpreting expression data across species and in disease. </p>
									         <p>Paper: <a href="https://genome.cshlp.org/content/29/1/53" target="_blank">A quantitative model for characterizing the evolutionary history of mammalian gene<br>&emsp;&emsp;&emsp;&ensp;expression</a>. <em>Genome Research</em>, Jan 2019. </p>
            
									         </div>
									    </div>
									</div>
									<div class="panel panel-default">
									    <div class="panel-heading" role="tab" id="headingThree">
									        <h4 class="panel-title">
									            <a class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">Long noncoding RNA evolution
									            </a>
									        </h4>
									    </div>
									    <div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
									        <div class="panel-body">
									        <p><img src="images/lncrnas.png"></p>
									         <p>The development of RNA-sequencing technology led to the discovery that thousands, of long noncoding RNAs (lncRNAs) are pervasively transcribed in the mammalian genome. Yet, the function of these genes are unknown. In a collaboration with <a href="http://garberlab.umassmed.edu/" target="_blank">Manuel Garber (UMass Medical)</a>, we developed a computational tool, <a href="http://slncky.github.io/" target="_blank"><em>slncky</em></a>, that identifies lncRNAs from RNA-sequencing data and examines the evolutionary history of these genes. Our tool discovers orthologous lncRNAs and calculates metrics relevant for noncoding transcript evolution such as transcript-transcript identity (what percentage of the transcript sequence is transcribed in another species) and splice site conservation. Our analysis revealed distinct patterns of selection, uncovering several classes of lncRNAs that likely each have distinct functions.  Out of the tens and thousands of currently annotated lncRNAs, we identify 233 constrained lncRNAs which are browsable through the <a href="https://slncky.umassmed.edu/" target="_blank"><em>slncky</em> Evolution Browser</a>.</p>
									         <p>Paper: <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0880-9" target="_blank">Evolutionary analysis across mammals reveals distinct classes of long non-coding RNAs</a>.<br>&emsp;&emsp;&emsp;<em>Genome Biology</em>, January 2016. </p>
									        </div>
									    </div>
									</div>
									
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>

			<section class="colorlib-experience" data-section="education">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<h2 class="colorlib-heading animate-box">Education</h2>
						</div>
					</div>
					<div class="row">
						<div class="col-md-12">
				         <div class="timeline-centered">

					         <article class="timeline-entry animate-box" data-animate-effect="fadeInLeft">
					            <div class="timeline-entry-inner">

					               <div class="timeline-icon color-3">
					                  <i class="icon-pen2"></i>
					               </div>

					               <div class="timeline-label">
					                  <h2><a href="#">PhD in Bioinformatics and Integrative Genomics, Division of Health Sciences and Technology, MIT</a> <span>2011-2018</span></h2>
					                  <p><b>Thesis:</b> Evolutionary signatures for unearthing functional elements in the human transcriptome
									  <br><b>Faculty advisor:</b> Aviv Regev
					               </div>
					            </div>
					         </article>


					         <article class="timeline-entry animate-box" data-animate-effect="fadeInRight">
					            <div class="timeline-entry-inner">
					               <div class="timeline-icon color-1">
					                  <i class="icon-pen2"></i>
					               </div>
					               <div class="timeline-label">
					               	<h2><a href="#">BS/MS in Biomedical Computation, Stanford University</a> <span>2006-2011</span></h2>
					                  <p><b>Thesis:</b> Analysis of Transcription Factor ChIP-Seq Datasets 
									  <br><b>Faculty advisor:</b> Gill Bejerano
					               </div>
					            </div>
					         </article>

					      </div>
					   </div>
				   </div>
				</div>
			</section>

			<section class="colorlib-contact" data-section="contact">
				<div class="colorlib-narrow-content">
					<div class="row">
						<div class="col-md-6 col-md-offset-3 col-md-pull-3 animate-box" data-animate-effect="fadeInLeft">
							<span class="heading-meta">Contact</span>
							<h2 class="colorlib-heading">Get in touch</h2>
						</div>
					</div>
					
					<div class="row">
						<div class="col-md-5">
							<div class="colorlib-feature colorlib-feature-sm animate-box" data-animate-effect="fadeInLeft">
								<div class="colorlib-icon">
									<i class="icon-mail"></i>
								</div>
								<div class="colorlib-text">
									<p>jennifer underscore chen at fas dot harvard dot edu</p>
								</div>
							</div>

							<div class="colorlib-feature colorlib-feature-sm animate-box" data-animate-effect="fadeInLeft">
								<div class="colorlib-icon">
									<i class="icon-map"></i>
								</div>
								<div class="colorlib-text">
									<p>Rm 1008, 16 Divinity Avenue <br> Cambridge, MA</p>
								</div>
							</div>
						</div>
					</div>
				</div>
			</section>

		</div><!-- end:colorlib-main -->
	</div><!-- end:container-wrap -->
	</div><!-- end:colorlib-page -->

	<!-- jQuery -->
	<script src="js/jquery.min.js"></script>
	<!-- jQuery Easing -->
	<script src="js/jquery.easing.1.3.js"></script>
	<!-- Bootstrap -->
	<script src="js/bootstrap.min.js"></script>
	<!-- Waypoints -->
	<script src="js/jquery.waypoints.min.js"></script>
	<!-- Flexslider -->
	<script src="js/jquery.flexslider-min.js"></script>
	<!-- Owl carousel -->
	<script src="js/owl.carousel.min.js"></script>
	<!-- Counters -->
	<script src="js/jquery.countTo.js"></script>
	
	
	<!-- MAIN JS -->
	<script src="js/main.js"></script>

	</body>
</html>

